hack/install-e2e-dependencies.sh

```diff
@@ -6,30 +6,14 @@ set -eux
 SCRIPT_DIR="$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
 source "${SCRIPT_DIR}/libe2e.sh"
 
-curl -Lo helm.tar.gz \
-     https://storage.googleapis.com/kubernetes-helm/helm-v2.6.1-linux-amd64.tar.gz
-tar xvf helm.tar.gz
-sudo mv linux-amd64/helm /usr/local/bin
-
-curl -Lo kubectl \
-     https://storage.googleapis.com/kubernetes-release/release/$KUBERNETES_VERSION/bin/linux/amd64/kubectl
-chmod +x kubectl
-sudo mv kubectl /usr/local/bin/
-
-curl -Lo minikube \
-     https://storage.googleapis.com/minikube/releases/v0.23.0/minikube-linux-amd64
-chmod +x minikube
-sudo mv minikube /usr/local/bin/
-
-docker run -v /usr/local/bin:/hostbin quay.io/jetstack/ubuntu-nsenter cp /nsenter /hostbin/nsenter
```

Comment:
❓ I see all this stuff gets installed here: https://github.com/jetstack/test-infra/blob/master/images/minikube-in-go/Dockerfile
Great, that should speed things up and save some network traffic. 
* How easy is it to get launch a KVM VM from inside a Docker container?
* How do you ensure that we're using the correct version of `kubectl`, if it is installed there but `minikube start --kubernetes-version...` is specified here?

Reply:
The KVM VM is launched in a container by passing through the libvirt socket into the container. Anything that works with libvirt *should* work okay within a container too.

The `KUBERNETES_VERSION` environment variable is set as part of the minikube-in-go docker image, which helps ensure we use the correct version of kubectl with kubernetes 😄